_datascience/cbioportal.md

```diff
@@ -1 +1,34 @@
+---
+title: cBioPortal
+primary_reviewers: vortexing, abbywall
+---
+
+[**cBioPortal**](https://www.cbioportal.org/) is an open-access, open-source resource for interactive exploration of multidimensional cancer genomics data sets developed by Memorial Sloan Kettering (MSK). The Data Science Lab (DaSL) in conjunction with the Scientific Computing Group (SciComp) at Fred Hutch have integrated a Fred Hutch instance of cBioPortal for research use within the Fred Hutch. 
+
+The Fred Hutch deployment of cBioPortal allows users to:
+
+- 
+- 
+- 
+
+## Getting Started
+
+### Training and Courses
+- Find out more about using the Fred Hutch cBioPortal [How-To in the DaSL Resource Library]()
+
+### Technical Documentation and Repositories
+- 
```

Comment:
Learn how to prepare your data files for upload into cBioPortal, launch a local instance of cBioPortal and how to use some publicly available tools to prepare your data in [this repository]().

Reply:
Same for this review what Taylor updated